pkg/operator/configobserver/cloudprovider/observe_cloudprovider.go

```diff
@@ -66,8 +68,19 @@ func (c *cloudProviderObserver) ObserveCloudProviderNames(genericListers configo
 		return existingConfig, append(errs, err)
 	}
 
+	external, err := c.isCloudProviderExternal(listers, infrastructure.Status.Platform, recorder)
```

Comment:
nit: why you pass recorder here?

Reply:
I think this is a leftover from earlier, will remove it.